src/test/daml/Daml/Finance/Settlement/Test/Transfer.daml

```diff
@@ -316,7 +326,84 @@ run4 = script do
 
   pure ()
 
+-- Transfer of commercial bank money from Alice to Charlie via Bob (acting as an intermediary/passthrough).
+-- +---------------------------------------------------+
+-- | Accounts                                          |
+-- +---------------------------+-----------------------+
+-- | structure:                | used for:             |
+-- +---------------------------+-----------------------+
+-- |           Bank            |                       |
+-- |      /     |    \         | commercial bank money |
+-- | Sender -> CCP -> Receiver |                       |
+-- +---------------------------+-----------------------+
+run5 : Script ()
+run5 = script do
+  TestParties{..} <- setupParties
+
+  -- Account and holding factory
+  let fp = [("FactoryProvider", singleton publicParty)]
+  accountFactoryCid <- toInterfaceContractId <$> Account.createFactory provider fp
```

Comment:
Given that all accounts are at the bank, do we need a `provider` at all? The bank could provide the account and holding factories.